code/modules/mob/living/simple_animal/borer/egg.dm

```diff
@@ -98,6 +98,9 @@
 	else
 		..()
 
+/obj/item/weapon/reagent_containers/food/snacks/borer_egg/attack_ghost(var/mob/dead/observer/O)
+	visible_message(message = "<span class='notice'>\The [src] wriggles vigorously.</span>", blind_message = "<span class='danger'>You hear what you think is someone jiggling a jelly.</span>")
```

Comment:
None of the "ping" functions have an extra cooldown, I do believe that because it's an "attack" it already has a cooldown by itself. And besides, if you chatspam with an object it's ding-dong bannu anyway.